pkgs/by-name/sa/sasquatch/package.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  lz4,
+  lzo,
+  which,
+  xz,
+  zlib,
+  zstd,
+  bigEndian ? false,
+}:
+
+let
+  drv = stdenv.mkDerivation rec {
+    pname = "sasquatch";
+    version = "4.5.1-4";
+
+    src = fetchFromGitHub {
+      owner = "onekey-sec";
+      repo = pname;
+      rev = "${pname}-v${version}";
```

Comment:
```suggestion
      repo = "sasquatch";
      rev = "sasquatch-v${finalAttrs.version}";
```

Reply:
Nice idea to use fixed-point attribute here, but then shouldn't `hash` also come from an input argument? :thinking:

Reply:
No. My proposals should make the build working correctly.

Reply:
I understand, but IMO `overrideAttrs (_: { version = "foo"; })` will lead to surprising results, as it will affect src fetching

Reply:
Am I overthinking?

Reply:
Uhm dunno... feel free to not merge the commit suggestion.

Reply:
Checked that there are many examples in nixpkgs already, so it is fine by me